listenbrainz_spark/ftp/download.py

```diff
@@ -119,15 +135,5 @@ def download_artist_relation(self, directory, artist_relation_dump_id=None):
             Returns:
                 dest_path (str): Local path where artist relation has been downloaded.
         """
-        self.connection.cwd('/pub/musicbrainz/listenbrainz/labs/artist-credit-artist-credit-relations/')
-        artist_relation_dump = self.list_dir()
-        req_artist_relation_dump = self.get_req_dump(artist_relation_dump, artist_relation_dump_id, 5)
-
-        self.connection.cwd(req_artist_relation_dump)
-        artist_relation_file_name = self.get_file_name(req_artist_relation_dump)
-
-        t0 = time()
-        current_app.logger.info('Downloading {} from FTP...'.format(artist_relation_file_name))
-        dest_path = self.download_dump(artist_relation_file_name, directory)
-        current_app.logger.info('Done. Total time: {:.2f} sec'.format(time() - t0))
+        dest_path = self.get_spark_dump_path(directory, artist_relation_dump_id, config.FTP_ARTIST_RELATION_DIR, 5)
```

Comment:
5?

Reply:
`artist-credit-artist-credit-relations-01-20191230-134806/`: dump identifier position.

Reply:
There has to be some way to identify the dump id in the dump name. I cannot think of any other way. The identifier position will always be constant for a given function until we suddenly decide to change all of our dump names. Also, passing the identifier make it possible for a single function to download the different dumps.  If I am not passing the identifier position I will have to duplicate the following line of code in every function:

`dump_id = dump_name.split('-')[ identifier position ]`

if there is a better way out, I would love to know :)

Reply:
Generally, I'd like to use constants instead of magic numbers.

Reply:
Like

```
PI = 3.14
print(PI)
```

Instead of 
```
print(3.14)
```